correct grammatical errors (#547)
spidernet-io/spiderpool: docs/develop/changelog.md

# Changelog

How to automatically generate changelogs:

1. All PRs should be labeled with "pr/release/***" and can be merged.

> 2. When pushing a tag, the changelog will be created automatically

Reviewer: ```suggestion
2. When you add the label, the changelog will be created automatically.
```
